missilewars-plugin/src/main/java/de/butzlabben/missilewars/commands/UserCommands.java

```diff
@@ -95,44 +95,43 @@ public void changeCommand(CommandSender sender, String[] args) {
         Player player = (Player) sender;
         
         if (args.length < 1) {
-            player.sendMessage(Messages.getPrefix() + "§cNumber needed.");
+            player.sendMessage(Messages.getMessage("command.team_number_needed"));
             return;
         }
 
         if (args.length > 1) {
-            player.sendMessage(Messages.getPrefix() + "§cToo many arguments.");
+            player.sendMessage(Messages.getMessage("command.to_many_arguments"));
             return;
         }
 
         Game game = GameManager.getInstance().getGame(player.getLocation());
         if (game == null) {
-            player.sendMessage(Messages.getMessage("not_in_arena"));
+            player.sendMessage(Messages.getMessage("game.not_in_game_area"));
             return;
         }
-
+        
         if (game.getState() != GameState.LOBBY) {
-            player.sendMessage(Messages.getPrefix() + "§cThe game is not in the right state to change your team right now");
+            player.sendMessage(Messages.getMessage("team.change_team_not_now"));
             return;
         }
 
         try {
             MWPlayer mwPlayer = game.getPlayer(player);
             int teamNumber = Integer.parseInt(args[0]);
             Team to = teamNumber == 1 ? game.getTeam1() : game.getTeam2();
-            int otherCount = to.getEnemyTeam().getMembers().size() - 1;
-            int toCount = to.getMembers().size() + 1;
-            int diff = toCount - otherCount;
-            if (diff > 1) {
```

Comment:
why did we remove this logic?

Reply:
All checks are always included.

- argument is "1" or "2"
- fair team switch via `if (to != game.getNextTeam()) { ... }` so that you can't change to the already bigger team